src/components/button-more/popup/popup.js

```diff
@@ -0,0 +1,20 @@
+import angular from "angular";
+import ngMaterial from "angular-material";
+import template from "./popup.html";
+
+export class PopupController {
+  constructor() {}
+  handleItemClickEdit() {
```

Comment:
popup menu isn't "smart". We will make logic in accounts/tags/transactions components, so handler for editing/deleting should be there. 
it Is my opinion. what do you think?